cypress/e2e/rooms/duplicateRoom.feature

```diff
@@ -1,47 +1,62 @@
-@unstable_test
-# feature is not yet implemented
+@stable_test
+@regression_test
 Feature: Room - To duplicate the existing room
 
     As a teacher I want to duplicate an existing room, so that I can have a copy of it.
 
     Scenario: Teacher duplicates and deletes the room, including pre & post conditions where applicable
         Given I am logged in as a '<teacher>' at '<namespace>'
+        Given I am logged in as a '<admin>' at '<namespace>'
 
-        # pre-condition: teacher creates a new room
+        # pre-condition: teacher creates a new room with multi and single coulmn board in it
+        Given admin enables video conference for the school in the school settings page
+        Given I am logged in as a '<teacher>' at '<namespace>'
         Given a room named '<room_name>' exists
-
-        # teacher is able duplicate room
+        Given a multi-column board named '<board_title>' exists in the room
+        Given multi column board is published to not to be in a draft mode
+        Given the multi-column board has a column with a card
+        Given video conference is added in the card
+        Given etherpad is added in the card
```

Comment:
both video and etherpad involve 3rd party services
a simple board containing for example a rich text element would be also ok for this case, and probably faster / more stable

Reply:
I added now Link element instead BBB & Epad.